image-carousel.html

```diff
@@ -172,6 +172,10 @@
      */
 
      class ImageCarousel extends Polymer.GestureEventListeners(Polymer.Element) {
+      constructor() {
+        super();
+        this._slotChangeHandler = this.computeNumberOfImages.bind(this);
```

Comment:
Why is this required? why not use `computeNumberOfImages` directly in `eventListener`?

Reply:
The problem is in that the disconnectedCallback() requires a reference to the function as well, otherwise you'll have a memory leak.  Using
```
removeEventListener('slotchange', this.computeNumberOfImages.bind(this));
```
will not work, since
```
this.computeNumberOfImages.bind(this) !== this.computeNumberOfImages.bind(this)
```
trust me --- I tried that in a previous commit :)